### PR TITLE
Bugfix/nachimehta subscription history

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
**Are you a current Fivetran customer?** 

Fivetran created PR built from the contributions of @nachimehta

**What change(s) does this PR introduce?** 

This PR adds documentation that highlights the `subscription_history` change that was introduced within the [Stripe Source PR](https://github.com/fivetran/dbt_stripe_source/pull/37). 

Additionally, the main change within this PR will be the adjustment of the source package dependency.

**Did you update the CHANGELOG?** 

- [X] Yes

**Does this PR introduce a breaking change?**

- [X] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

Since the source PR will be a breaking change, this will also be breaking.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [X] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [X] Yes, Issue/Feature [dbt_stripe_source #37]((https://github.com/fivetran/dbt_stripe_source/pull/37))
- [ ] No 

**How did you test the PR changes?** 

 

- [X] CircleCi  
- [X] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [X] BigQuery
- [X] Redshift
- [X] Snowflake
- [X] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
💇‍♀️ 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/stripe_source
-  version: [">=0.5.0", "<0.6.0"]
+# - package: fivetran/stripe_source
+#   version: [">=0.5.0", "<0.6.0"] 
+  - git: https://github.com/fivetran/dbt_stripe_source.git
+    revision: bugfix/nachimehta-subscription-history
+    warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]: To be changed before release to the latest source package version (`v0.7.0` range)

**Comment by fivetran-joemarkiewicz** [comment]
This can be tested using the following `packages.yml` config.

```yml
packages:
  - git: https://github.com/fivetran/dbt_stripe.git
    revision: bugfix/nachimehta-subscription-history
    warn-unpinned: false
```

**Review by fivetran-reneeli — COMMENTED** [review]
Looks good!

File: packages.yml (hunk 1 above)
Comment (fivetran-joemarkiewicz) [inline packages.yml:6]:
```suggestion
 - package: fivetran/stripe_source
   version: [">=0.5.0", "<0.6.0"] 

```

File: packages.yml [hunk 2]
```diff
@@ -1,3 +1,3 @@
 packages:
-- package: fivetran/stripe_source
-  version: [">=0.5.0", "<0.6.0"]
+ - package: fivetran/stripe_source
+   version: [">=0.5.0", "<0.6.0"]
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:3]:
```suggestion
 - package: fivetran/stripe_source
   version: [">=0.6.0", "<0.7.0"] 
```

**Pull request merged**